### PR TITLE
Reverse Supercell Mapping

### PR DESCRIPTION
**Pull request by hmlli**
Adding function to map SC onto UC positions

@jmmshn check tests and function

### CODE REVIEW

File: pymatgen_diffusion/neb/full_path_mapper.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ class MigrationGraph(MSONable):
     def __init__(
         self,
         structure: Structure,
-        migration_graph: StructureGraph,
+        m_graph: StructureGraph,
```
Comment (hmlli) [inline pymatgen_diffusion/neb/full_path_mapper.py:82]: Renamed to avoid confusion with the main Class

**Comment by shyuep** [comment]
Pls hold off on further development for a few hours. I am implementing a major reorganization of pymatgen-diffusion. Once I am done, you can pull in the master branch and continue working.

**Comment by shyuep** [comment]
I am done with my rewrite. Feel free to pull main master into your PR.

**Comment by hmlli** [comment]
@shyuep This is now ready to merge, please check and merge. Thanks!

**Comment by shyuep** [comment]
Thanks. I will soon add a contributor page so that we can acknowledge these contributions.

**Pull request merged**